cfn-resources/project/mongodb-atlas-project.json

```diff
@@ -73,6 +73,16 @@
     "Roles": {
       "type": "string",
       "description": "One or more organization- or project-level roles to assign to the MongoDB Cloud user.Items Enum: \"GROUP_CLUSTER_MANAGER\" \"GROUP_DATA_ACCESS_ADMIN\" \"GROUP_DATA_ACCESS_READ_ONLY\" \"GROUP_DATA_ACCESS_READ_WRITE\" \"GROUP_OWNER\" \"GROUP_READ_ONLY\""
+    },
+    "Tags": {
+      "type": "object",
+      "description": "Map that contains key values between 1 to 255 characters in length for tagging and categorizing the project. To learn more, see https://www.mongodb.com/docs/atlas/tags/",
+      "patternProperties": {
+        "^.*$": {
```

Comment:
Better to use a `"^.*{1, 255}$"` perhaps?

Reply:
not a strong opinion, but similar to other discussions, maybe don't do any check here and let server fails if param is not ok